components/collapsible/showmore/src/index.js

```diff
@@ -0,0 +1,52 @@
+import React, {useState} from 'react'
+import PropTypes from 'prop-types'
+import Button from '@s-ui/react-atom-button'
+import cx from 'classnames'
+
+const CollapsibleShowmore = ({
+  children,
+  copy = 'Show more',
+  maxHeight = 100
+}) => {
+  const baseClass = 'sui-CollapsibleShowmore'
+  const [heightContainer, setHeightContainer] = useState(maxHeight)
+  const [hideButton, setHideButton] = useState(false)
+
+  const buttonClassName = cx(`${baseClass}-actionButton`, {
+    'is-hidden': hideButton
+  })
+
+  const unfold = () => {
+    setHeightContainer('auto')
```

Comment:
The `setHeightContainer` state seems to be unnecessary as could be calculated from the `hideButton` state.